README.md

```diff
@@ -1,2 +1,27 @@
-# pytest-browserstack
-https://browserstack.atlassian.net/wiki/spaces/PROD/pages/3358687296/Selenium+Frameworks+Documentation#Git-Repo-Names
+# PyTest with Browserstack
+
+PyTest Integration with BrowserStack.
+
+![BrowserStack Logo](https://d98b8t1nnulk5.cloudfront.net/production/images/layout/logo-header.png?1469004780)
+
+## Setup
+
+* Clone the repo
+* Install dependencies `pip install -r requirements.txt`
+* To run your automated tests using BrowserStack, you must provide a valid username and access key. This can be done either by using a .browserstack configuration file in the working directory or your home directory, or by setting the BROWSERSTACK_USERNAME and BROWSERSTACK_ACCESS_KEY environment variables.
+
+## Running your single tests
+* To run single tests, run `paver run single`
+
+## Running your local tests
+* To run a local test, first go to tests/local_test/conftest.py then edit BROWSERSTACK_ACCESS_KEY on line 8
```

Comment:
```suggestion
* To run a local test, (if you have not set the BROWSERSTACK_ACCESS_KEY environment variable) go to tests/local_test/conftest.py then replace YOUR_ACCESS_KEY on line 8 with your Browserstack Access Key.
```
This could be a bit ambiguous, as there are multiple BROWSERSTACK_ACCESS_KEY on the line being referred to..
Also need not do this step if env variable is set.

Reply:
fixed https://github.com/browserstack/pytest-browserstack/pull/2/commits/cf0b5bb4f735d42956c6165eca7934c9000dada3